app/views/events/settings/_features.html.erb

```diff
@@ -1,13 +1,15 @@
 <%# locals: (event:) %>
 
-<%= render partial: "features/preview", locals: {
-      id: "home-page-redesign",
-      classes: ["feature-home-page-redesign"],
-      feature_flag: :event_home_page_redesign_2024_09_21,
-      name: "Home Page Redesign",
-      description: "A fresh new look for your event's home page!",
-      event:
-    } if Flipper.enabled?(:event_home_page_redesign_2024_09_21, event) %>
+<% admin_tool("mb3") do %>
```

Comment:
use `admin_tool_if` so we still show it if it is enabled. so `admin_tool_if` if it is not enabled.